.php_cs.cache

```diff
@@ -0,0 +1 @@
+{"php":"8.0.3","version":"2.18.3","indent":"    ","lineEnding":"\n","rules":{"blank_line_after_namespace":true,"braces":true,"class_definition":true,"constant_case":true,"elseif":true,"function_declaration":true,"indentation_type":true,"line_ending":true,"lowercase_keywords":true,"method_argument_space":{"on_multiline":"ensure_fully_multiline","keep_multiple_spaces_after_comma":true},"no_break_comment":true,"no_closing_tag":true,"no_spaces_after_function_name":true,"no_spaces_inside_parenthesis":true,"no_trailing_whitespace":true,"no_trailing_whitespace_in_comment":true,"single_blank_line_at_eof":true,"single_class_element_per_statement":{"elements":["property"]},"single_import_per_statement":true,"single_line_after_imports":true,"switch_case_semicolon_to_colon":true,"switch_case_space":true,"visibility_required":true,"encoding":true,"full_opening_tag":true,"array_syntax":{"syntax":"short"},"ordered_imports":{"sortAlgorithm":"alpha"},"no_unused_imports":true,"not_operator_with_successor_space":true,"trailing_comma_in_multiline_array":true,"phpdoc_scalar":true,"unary_operator_spaces":true,"binary_operator_spaces":true,"blank_line_before_statement":{"statements":["break","continue","declare","return","throw","try"]},"phpdoc_single_line_var_spacing":true,"phpdoc_var_without_name":true},"hashes":{"src\/ViewModelsServiceProvider.php":2278103137,"src\/ViewModel.php":894200466,"src\/Console\/ViewModelMakeCommand.php":3721917699,"tests\/ViewModelTest.php":2153730530,"tests\/ViewModelMakeCommandTest.php":1388068081,"tests\/TestCase.php":4131197707,"tests\/DummyViewModel.php":3408407484}}
```

Comment:
Please add `.php_cs.cache` to your global gitignore and remove it:

```
git rm .php_cs.cache
git commit …
```

Reply:
@brendt This file exists in the gitignore in [`spatie/package-skeleton-laravel`](https://github.com/spatie/package-skeleton-laravel/blob/master/.gitignore#L3) - would you prefer I merge that `.gitignore` into this package's `.gitiginore`?